npm/private/test/snapshots/next_links_defs.bzl

```diff
@@ -89,49 +81,64 @@ def npm_imported_package_store_internal():
             ":.aspect_rules_js/node_modules/json5@2.2.3/pkg": "json5",
             ":.aspect_rules_js/node_modules/lru-cache@5.1.1/pkg": "lru-cache",
             ":.aspect_rules_js/node_modules/ms@2.1.3/pkg": "ms",
-            ":.aspect_rules_js/node_modules/nanoid@3.3.11/pkg": "nanoid",
-            ":.aspect_rules_js/node_modules/next@15.2.4_1790925128/pkg": "next",
             ":.aspect_rules_js/node_modules/node-releases@2.0.27/pkg": "node-releases",
-            ":.aspect_rules_js/node_modules/picocolors@1.1.1/pkg": "picocolors",
-            ":.aspect_rules_js/node_modules/postcss@8.4.31/pkg": "postcss",
-            ":.aspect_rules_js/node_modules/react-dom@19.2.0_react@19.2.0/pkg": "react-dom",
-            ":.aspect_rules_js/node_modules/react@19.2.0/pkg": "react",
-            ":.aspect_rules_js/node_modules/scheduler@0.27.0/pkg": "scheduler",
             ":.aspect_rules_js/node_modules/semver@6.3.1/pkg": "semver",
             ":.aspect_rules_js/node_modules/semver@7.7.3/pkg": "semver",
             ":.aspect_rules_js/node_modules/sharp@0.33.5/pkg": "sharp",
             ":.aspect_rules_js/node_modules/simple-swizzle@0.2.4/pkg": "simple-swizzle",
-            ":.aspect_rules_js/node_modules/source-map-js@1.2.1/pkg": "source-map-js",
-            ":.aspect_rules_js/node_modules/streamsearch@1.1.0/pkg": "streamsearch",
-            ":.aspect_rules_js/node_modules/styled-jsx@5.1.6_324456147/pkg": "styled-jsx",
-            ":.aspect_rules_js/node_modules/tslib@2.8.1/pkg": "tslib",
             ":.aspect_rules_js/node_modules/update-browserslist-db@1.1.4_browserslist@4.27.0/pkg": "update-browserslist-db",
             ":.aspect_rules_js/node_modules/yallist@3.1.1/pkg": "yallist",
         } | select({
             "@aspect_rules_js//platforms/pnpm:darwin_arm64": {
+                ":.aspect_rules_js/node_modules/@img+sharp-darwin-arm64@0.33.5/pkg": "@img/sharp-darwin-arm64",
```

Comment:
This demos the main goal + fix

Reply:
Note that nextjs depends on babel which has a circular dep, so even after https://github.com/aspect-build/rules_js/pull/2559 this fix is still relevant for nextjs.